cypress/integration/non-regression-tests-suite/event-signup-flow.spec.js

```diff
@@ -82,14 +79,53 @@ describe("Feature - Event sign up : Tests execution date and time : " + new Date
 			}
 		});
 	});
+	it("It shows the Sign up complete message - for existing candidate", function () {
+		cy.wait(5000);
```

Comment:
It may be worth extracting this to a method to make it clearer as to why we need to wait? So something like `waitForRegistrationToComplete()`?

Reply:
Resolved